soccer/CMakeLists.txt

```
@@ -172,6 +173,9 @@ set(REFEREE_NODE_DEPS_LIBRARIES robocup)
 set(RADIO_NODE_DEPS_SYSTEM_LIBRARIES)
 set(RADIO_NODE_DEPS_LIBRARIES robocup)
 
+set(SOCCER_MOM_NODE_DEPS_SYSTEM_LIBRARIES)
+set(SOCCER_MOM_DEPS_LIBRARIES robocup)
```

```suggestion
set(SOCCER_MOM_NODE_DEPS_LIBRARIES robocup)
```

here's the typo you're looking for. note line 230 expects the variable to be named as such.